content/blog/2020-05-04-dvc-3-years-and-1-0-release.md

```diff
@@ -0,0 +1,198 @@
+---
+title: DVC 3 Years Anniversary and 1.0 Pre-release
+date: 2020-05-04
+description: |
+  Today, we’ve got three big things to announce: 🎉 3rd-year anniversary of DVC,
+  🚀 DVC 1.0 pre-release is ready and ⭐ 5000 GitHub starts.
+
+descriptionLong: |
+  Today, we’ve got three big things to announce.
+
+  🎉 3rd-year anniversary of DVC
+
+  🚀 DVC 1.0 pre-release is ready
+
+  ⭐ DVC has reached 5K GitHub starts (coincidently on the same day)
+
+  We are sharing our learnings from this journey and how they affected the new
+  DVC 1.0 release.
+picture: 2020-05-04/5k_stars.png
+pictureComment: 5000 GitHub stars
+author: dmitry_petrov
+commentsUrl: https://discuss.dvc.org/t/dvc-3-years-anniversary-and-1-0-pre-release/374
+tags:
+  - Release
+  - MLOps
+  - DataOps
+  - CI/CD
+---
+
+## 3 years anniversary!
+
+3 years ago on **May 4th, 2017** the first DVC blog post was published
+[Data Version Control beta release: iterative machine learning](https://blog.dataversioncontrol.com/data-version-control-beta-release-iterative-machine-learning-a7faf7c8be67).
+It was the first DVC tutorial and later we made a redirect to the new tutorial
+page to not to confuse users.
+[The first DVC discussion on Reddit](https://www.reddit.com/r/Python/comments/698ian/dvc_data_scientists_collaboration_and_iterative/).
+A few days later it was
+[republished on other sources](https://www.kdnuggets.com/2017/05/data-version-control-iterative-machine-learning.html)
+
+Today, DVC gets recognized at professional conferences: people spot our logo,
+and sometimes even our faces, and want to chat. There's much more content about
+DVC coming from bloggers than from inside our organization. We're seeing more
+and more job postings that list DVC as a requirement, and we're showing up in
+[data science textbooks](https://www.amazon.com/Learn-Python-Building-Science-Applications/dp/1789535360).
+When we find a new place DVC is mentioned, we celebrate in our Slack - we've
+come a long way!
+
+The data science and ML space is fast-paced and vibrant, and we're proud that
+DVC is making an impact on discussions about best practices for healthy,
+sustainable ML. Every week, we chat with companies and research groups using DVC
+to make their teams more productive. We're proud to be part of the growing MLOps
+movement: so far, a majority of CI/CD for ML projects are implemented with DVC
+under the hood.
+
+I can confidently say that DVC wouldn't have been possible without a lot of help
+from our community. Thank you to everyone who has supported us:
+
+**DVC core team.** The core team of the project takes the majority of the
```

Comment:
I think take a bigger scope- mention that the team is now 12 full-time people or however large we technically are (even though they're not all "core DVC engineering"; this is more about Iterative).